BlockEntity/BEFruitpress.cs

```diff
@@ -93,6 +93,8 @@ BlockEntityAnimationUtil animUtil
 
         long listenerId;
         double juiceableLitresCapacity = 10;
+        double screwPercent;
+        double pressSqueezeRel;

```

Comment:
An internal version of squeezeRel for comparison to make sure the crank is touching the mash before it activates.